app/scripts/modules/core/src/pipeline/config/stages/evaluateVariables/EvaluateVariablesStageConfig.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { map, set } from 'lodash';
+
+import { IStageConfigProps, StageConfigField, MapEditor } from 'core';
+
+export interface IEvaluateVariablesStageConfigState {
```

Comment:
I don't see this state being used anywhere?

Reply:
doh! forgot to remove the types after I chucked out state completely.

nice catch. thanks!